tests/test_cli/test_generate_wealth.py

```diff
@@ -104,6 +104,12 @@ def test_wealth_commands(self):
 
         self.generate_wealth()
 
+
+class TestWealthCommandsNegative(AEATestCaseMany):
+    """Test case for CLI wealth commands, negative case."""
+
+    @pytest.mark.integration
+    @pytest.mark.flaky(reruns=MAX_FLAKY_RERUNS_INTEGRATION)
     def test_wealth_commands_negative(self):
```

Comment:
This is not integration! only the positive one is

Reply:
ok sorry